incubator/spark-history-server/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+{{- if .Values.createNamespace }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace }}
+{{- end }}
```

Comment:
Helm does the namespace handling to create a namespace. Is there a reason this routes around this to add an additional namespace? How is the Helm namespace creation and handling (e.g., `--namespace` flag`) not working?